NEWS.md

```diff
@@ -5,19 +5,43 @@ v0.9.0 (May 2?, 2019)
 ---------------------
 
 - Support for Julia v0.6 and v0.7 was dropped (#714, #717).
+- A `MOI.Utilities.Model` implementation of `ModelLike`, this should replace
+  most use cases of `MOI.Utilities.@model` (#781).
+- `add_constrained_variable` and `add_constrained_variables` were added (#759).
 - Support for indicator constraints was added (#709, #712).
+- `DualObjectiveValue` attribute was added (#473).
 - `RawParameter` attribute was added (#733).
+- A `Benchmarks` submodule was added to facilitate solver benchmarking (#769).
+- A `submit` function was added, this may for intance allow the user to submit
+  solutions or cuts to the solver from a callback (#775).
 - The field of `ObjectiveValue` was renamed to `result_index` (#729).
 - The `_constant` and `Utilities.getconstant` function were renamed to `constant`
-- The `Utilities.constant` function was renamed to `Utilities.constant_vector` (#740).
 - `REDUCTION_CERTIFICATE` result status was added (#734).
 - Abstract matrix sets were added (#731).
 - Testing improvements:
   * The testing guideline was updated (#728).
   * Quadratic tests were added (#697).
   * Unit tests for `RawStatusString`, `SolveTime`, `Silent` and `SolverName`
     were added (#726, #741).
-  * Existing tests had a few updates (#702, #703).
+  * A rotated second-order cone test was added (#759).
+  * A power cone test was added (#768).
+  * Tests for `ZeroOne` variables with variable bounds were added (#772).
+  * An unbounded test was added (#773).
+  * Existing tests had a few updates (#702, #703, #763).
+- Documentation improvements:
+  * Add a section on `CachingOptimizer` (#777).
```

Comment:
Nit: The structure of the sentences is inconsistent, e.g., "A power cone test was added" vs "Add a section on ...". I'd recommend "Added a section on ...". You could also ignore this since it's not a big deal.

Reply:
Indeed, we decided to use the past in `NEWS.md`